apps/web/components/OrderReturnProductCard/OrderReturnProductCard.vue

```diff
@@ -104,14 +106,63 @@
             </SfSelect>
           </label>
         </div>
+
+        <div class="my-2 w-full md:self-end">
+          <button
+            class="flex bg-gray-100 border border-gray-200 p-1 pl-4 w-full justify-between items-center"
+            :class="opened ? 'rounded-t-md' : 'rounded-md'"
+            @click="toggleDropdown"
+          >
+            {{ $t('account.ordersAndReturns.orderDetails.showDetails') }}
+            <SfIconChevronLeft
+              class="text-neutral-500 mr-2.5"
+              :class="{ 'rotate-90': opened, '-rotate-90': !opened }"
+            />
+          </button>
+          <div v-if="opened" class="dropdown">
+            <div v-for="(attribute, index) in orderGetters.getOrderAttributes(orderItem)" :key="index">
+              <h3
+                class="h-full w-full border border-gray-200 p-1.5"
+                :class="
+                  index === orderGetters.getOrderAttributes(orderItem).length - 1 &&
+                  orderGetters.getItemOrderProperties(orderItem).length === 0
+                    ? 'rounded-b-md'
+                    : ''
+                "
+              >
+                <div class="flex justify-between">
+                  <p class="text-sm px-2">{{ orderGetters.getOrderItemAttributeName(attribute) }}</p>
+                  <p class="text-sm px-2 text-gray-300">{{ orderGetters.getOrderItemAttributeValue(attribute) }}</p>
+                </div>
+              </h3>
+            </div>
+            <div v-for="(property, index) in orderGetters.getItemOrderProperties(orderItem)" :key="index">
+              <h3
+                class="h-full w-full border border-gray-200 p-1.5"
+                :class="index === orderGetters.getItemOrderProperties(orderItem).length - 1 ? 'rounded-b-md' : ''"
+              >
+                <div class="flex justify-between">
+                  <p class="text-sm px-2">{{ orderGetters.getItemOrderPropertyName(property) }}</p>
+                  <p class="text-sm px-2 text-gray-300">{{ orderGetters.getItemOrderPropertyValue(property) }}</p>
+                </div>
+              </h3>
+            </div>
+            <h3
+              v-if="orderGetters.getOrderAttributes(props.orderItem).length === 0"
```

Comment:
this should only be displayed if there are no properties and attributes available

Reply:
<img width="445" alt="Bildschirmfoto 2024-02-29 um 16 10 57" src="https://github.com/plentymarkets/plentyshop-pwa/assets/30629157/93c31c84-a9d8-4fce-84b6-39bcae52ab2a">